persist/storer.go

```diff
@@ -122,6 +124,10 @@ func NewFileStore(storagePath string, ilog log.Logger, ttl time.Duration) (Store
 		if err != nil {
 			ilog.Debugf(err.Error())
 		}
+
+		if err := store.deleteOldEntries(ttl); err != nil {
+			ilog.Debugf("Error deleting old entries in store file %q: %w. Ignoring", store.path, err)
+		}
```

Comment:
After looking at the code for a while I now realize we are deleting old entries that might have been created _on a previous run_. Do you think we could add a comment for this? Otherwise readers might be confused to not see `deleteOldEntries` being called at the end of the lifecycle.

Reply:
I actually re-think this and move it at the end on the save . This will give keys the chance to be updated in the current cycle.